Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "thoughtbot/Curry"
+github "thoughtbot/Curry" "tvos"
 github "thoughtbot/Runes" >= 3.0.0
```

Comment:
Correct. We don't need this to point at the `tvos` branch of Runes since we aren't building the targets.